ts/components/ui/FooterButtons.tsx

```diff
@@ -0,0 +1,95 @@
+import { Button, NativeBase, Text, View } from "native-base";
+import * as React from "react";
+import { StyleSheet } from "react-native";
+
+export enum FooterButtonsStyle {
+  HALF, // show inline buttons -  1/2 left, 1/2 right    -> [  b1  ] [  b2  ]
+  ONETHIRD, // show inline buttons - 1/3 left, 2/3 right -> [ b1 ] [   b2   ]
+  FULL // show stacked buttons - full size
+}
+
+type OwnProps = {
```

Comment:
Probably `ButtonExtraProps` would be better -- `OwnProps` inside `FooterButtons.tsx` looks like it would contain `FooterButtons` properties